frontend/public/locales/fr/status.json

```diff
@@ -48,5 +48,11 @@
     "hcaptcha": "https://www.hcaptcha.com/terms",
     "microsoft-data-privacy-policy": "https://www.microsoft.com/fr-ca/trust-center/privacy",
     "microsoft-service-agreement": "https://www.microsoft.com/en-ca/servicesagreement"
+  },
+  "form": {
+    "application-code-label": "Votre code de demande ou votre numéro du client (obligatoire)",
+    "application-code-description": "This code or number is found in the upper right-hand corner of the letter you received from Service Canada",
+    "sin-label": "Numéro d'assurance sociale (NAS) (obligatoire)",
```

Comment:
`(obligatoire)` should be removed since the InputField have the asterix and hidden `(obligatoire)` for a11y.